pkg/server/service_unix.go

```diff
@@ -72,3 +72,21 @@ func (c *criService) initPlatform() error {
 func (c *criService) cniLoadOptions() []cni.CNIOpt {
 	return []cni.CNIOpt{cni.WithLoNetwork, cni.WithDefaultConf}
 }
+
+// runningInUserNS detects whether we are currently running in a user namespace.
+// Originally copied from github.com/lxc/lxd/shared/util.go
+func runningInUserNS() bool {
+	uidmap, err := user.CurrentProcessUIDMap()
```

Comment:
Actually, given that (within scope of CRI itself) this is the only use of libcontainer/user; I'm wondering if we should simplify this to the original implementation in https://github.com/lxc/lxd/blob/master/shared/util.go#L690-L710

@AkihiroSuda wdyt?